examples/CMakeLists.txt

```diff
@@ -4,11 +4,8 @@ find_package(sdformat10 REQUIRED)
 
 set (CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++17")
 
-include_directories(${SDFormat_INCLUDE_DIRS})
-link_directories(${SDFormat_LIBRARY_DIRS})
-
 add_executable(simple simple.cc)
-target_link_libraries(simple ${SDFormat_LIBRARIES})
+target_link_libraries(simple sdformat10::sdformat10)
 
 add_executable(dom dom.cc)
-target_link_libraries(dom ${SDFormat_LIBRARIES})
+target_link_libraries(dom sdformat10::sdformat10)
```

Comment:
What if we set the SDF version (in this case, 10) to a variable instead? That way, if these changes were to be ported across different versions, updating the version would be easier since we would only need to update the variable's value.

Reply:
fixed in https://github.com/ignitionrobotics/sdformat/pull/780/commits/e1917735792647c8459c9a1c3cdcffec4ec3f5a3